cluster-operator/src/main/java/io/strimzi/operator/cluster/PlatformFeaturesAvailability.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2017-2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster;
+
+import io.vertx.core.json.Json;
+
+import java.util.Properties;
+
+public class PlatformFeaturesAvailability {
+    private Properties properties;
```

Comment:
Why are you using a `Properties` as a member? There's a fixed set of things you're interested in, so just use fields. Even if you had an unknown set of things using a `Map` is far better than using `Properties` because a `Map` with type arguments is much more typesafe than a `Properties`. The only reason to use a `Properties` is if you need to interact with an API which uses `Properties`.